src/Altinn.FileScan/Repository/BlobContainerClientProvider.cs

```diff
@@ -25,6 +26,7 @@ public class BlobContainerClientProvider : IBlobContainerClientProvider
         private readonly ILogger<IBlobContainerClientProvider> _logger;
```

Comment:
Just a question, shouldn't logger be used at some point in this file? 🤔 It seems `_logger` is declared here but not used anywhere. 🤔 Am I missing something? I see GitHub Actions also raised this issue. 🤔